Sources/Build/llbuild.swift

```diff
@@ -13,6 +13,21 @@ import Utility
 import PackageModel
 import PackageGraph
 
+/// An enum representing the different build options.
+public enum BuildOption {
+    /// Represents the option to build all targets and products (excluding test targets).
+    case allExcludingTests
+
+    /// Represents the option to build all targets and products (including test targets).
+    case allIncludingTests
+
+    /// Represents the option to build a specific target.
+    case target(ResolvedTarget)
+
+    /// Represents the option to build a specific product.
+    case product(ResolvedProduct)
```

Comment:
ambiguous to whom? llbuild's targets are never exposed to the outside world. The clients should assume they're passing in name of the targets/products in graph and then we validate the input against the data in graph.